identity-apps-core/features/org.wso2.identity.apps.authentication.portal.server.feature/resources/web.xml.j2

```diff
@@ -48,6 +48,12 @@
             <param-value>{{ downtime_banner.enabled }}</param-value>
         </context-param>
     {% endif %}
+    % if event.default_listener.validation.enable is defined %}
```

Comment:
Shouldn't we need an opening curly brace here?

```suggestion
    {% if event.default_listener.validation.enable is defined %}
```

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/7255/commits/3a7f06c65e19fc9443976e448086a1e39d08f280